ONSdigital/dp-frontend-filter-dataset-controller
# Hierarchy selection

### What

- Add hierarchy page rendering and mappings
- Call hierarchy api to get values to render
- Refactor handlers package so that each handler type is in its own file

### How to review

- Start up all usual services for CMD (including code-list api and hierarchy-api)
- Start up renderer on feature/hierarchy-selection
- Start up sixteens on feature/prevent-checkbox-label-overflow

Navigate to filter overview page, select goods-and-services. Verify you can navigate through the hierarchy successfully.

Note you will not be able to add or remove filter selections until the dataset api and filter api are available

### Who can review

Anyone
